raml/api.raml

```diff
@@ -58,6 +58,18 @@ mediaType: application/json
                 schema: schemas/404.json
                 example: !include response_examples/404/region.json
 
+    /security:
+      /regions:
+        get:
+          description: Return list of strings
+      /issues:
+        get:
+          description: Return list of Issue objects
+          uriParametrs:
+            region:
+              type: string
+              description: filter by region
+
```

Comment:
It would be very helpful to add a reference to response example (like at L29-34)